README.md

```diff
@@ -2,6 +2,8 @@
 
 This model is used to predict the risk of West Nile Virus being present two consecutive weeks in a row. 
 
+Most of the work in this project is data wrangling to get the data into a format where each row represents the results for a week / location collection, and the interval between the dates is regular (i.e. a week apart). Once this is accomplished, a model is fit that is very similar to a generalized linear model to predict the likelihood that WNV will be present if it was present last week. This allows our teams to go out and spray for mosquitos a week earlier than they would have otherwise.
```

Comment:
@tomschenkjr Thank you this is a lot better. Can you comment on the following:

> This project estimates the probability of

I would rather say "This project forecasts the risk of", but I don't really care that much. 

> a subsequent mosquito testing positive for West Nile Virus (WNV) when a mosquito has already been found to have WNV at a particular trap

This has been awkward to try and say, and I really like this wording.  My only issue is that the mosquito is singular, which makes it sound kind of like we somehow stop testing after finding the first mosquito. Do you think "any mosquito(s)" would be too awkward? (and add in "subsequent" in the first occurrence and change the verbs)

> multilevel (hierarchical) regression

Do you think we should cite the book here? I'm not sure about the formatting can we do footnotes in markdown or would an inline citation is appropriate?
Gelman, A., & Hill, J. (2007). *Data analysis using regression and multi-level/hierarchical models*. New York: Cambridge University Press.